hic.wdl

```diff
@@ -316,7 +316,8 @@ task align {
         source /opt/scripts/common/countligations.sh
         # Align reads
         echo "Running bwa command"
-        bwa mem -SP5M -t ${cpu} $reference_index_path ${fastqs[0]} ${fastqs[1]} | samtools view -hbS - > result.bam
+        bwa mem -SP5M -t ~{num_cpus} $reference_index_path ${fastqs[0]} ${fastqs[1]} |
```

Comment:
Good point, i did not consider that. I think the chip/atac approach makes sense, I will change so that only bwa uses multithreading and we don't overload the CPU.